sources/Libertinus_Keyboard_pse.ini

```diff
@@ -0,0 +1,17 @@
+[engkeys]
```

Comment:
I don’t like all these configuration files, they would be a maintenance nightmare. Would they are for? Can we do without them  or reduce them to one file or one per family?

Reply:
It was my expectation in designing the tool that it would be used with a per-family configuration file. And in fact that's how it is set up right now:

    $ grep FamilyName *.sfd
    LibertinusKeyboard-Regular.sfd:FamilyName: Libertinus Keyboard
    LibertinusMath-Regular.sfd:FamilyName: Libertinus Math
    LibertinusMono-Regular.sfd:FamilyName: Libertinus Mono
    LibertinusSans-Bold.sfd:FamilyName: Libertinus Sans
    LibertinusSans-Italic.sfd:FamilyName: Libertinus Sans
    LibertinusSans-Regular.sfd:FamilyName: Libertinus Sans
    LibertinusSerif-BoldItalic.sfd:FamilyName: Libertinus Serif
    LibertinusSerif-Bold.sfd:FamilyName: Libertinus Serif
    LibertinusSerifDisplay-Regular.sfd:FamilyName: Libertinus Serif Display
    LibertinusSerifInitials-Regular.sfd:FamilyName: Libertinus Serif Initials
    LibertinusSerif-Italic.sfd:FamilyName: Libertinus Serif
    LibertinusSerif-Regular.sfd:FamilyName: Libertinus Serif
    LibertinusSerif-SemiboldItalic.sfd:FamilyName: Libertinus Serif Semibold
    LibertinusSerif-Semibold.sfd:FamilyName: Libertinus Serif Semibold

    $ ls *.ini
    Libertinus_Keyboard_pse.ini Libertinus_Mono_pse.ini  
    Libertinus_Serif_Display_pse.ini Libertinus_Serif_pse.ini
    Libertinus_Math_pse.ini Libertinus_Sans_pse.ini
    Libertinus_Serif_Initials_pse.ini

The problem is that what constitutes a subjective family and what constitutes a family at the semantic level can differ.  However, it's also quite easy to set the tool up to look at a single configuration file, or to have some `.sfd`s have family-specific config files and others use the default. [Discussion continued at root-level.]